### PR TITLE
Hardware Page Revamp

### PR DESCRIPTION
**Pull request by yangshun**
Will add in more design fluff in future. Currently just a change in skin from the previous version.

screen shot 2015-05-15 at 9 53 08 am

### CODE REVIEW

**Comment by tgxworld** [comment]
@yangshun Lets squash the commits into one for this PR. Otherwise, we'll end up with a commit having the modified Docker image :smile_cat:

File: app/views/static_pages/hardware.html.haml [hunk 1]
```diff
@@ -1,58 +1,62 @@
...
+
+      %hr
+
+      %blockquote
+        %p
+          Our current bare metal server is sponsored by the team over at
+          = succeed '.' do
+            = link_to 'RubyTune.com', 'https://rubytune.com/', target: '_blank'
+
+          .sponsor-ruby-tune
```
Comment (tgxworld) [inline app/views/static_pages/hardware.html.haml]: I think we need to change this to `.sponsor-rubytune` in order for the image to display

**Comment by tgxworld** [comment]
> @yangshun Lets squash the commits into one for this PR. Otherwise, we'll end up with a commit having the modified Docker image

Opps I commented on the wrong PR xD

**Comment by yangshun** [comment]
Paiseh missed that out, thanks for catching it.

screen shot 2015-05-15 at 10 39 30 am

Lol I squashed both PRs anws :+1:

**Comment by tgxworld** [comment]
@yangshun Thanks!!

**Pull request merged**